external/urlfrontier/src/test/java/com/digitalpebble/stormcrawler/urlfrontier/URLFrontierContainer.java

```diff
@@ -28,7 +28,7 @@ public class URLFrontierContainer extends GenericContainer<URLFrontierContainer>
         implements URLFrontierContainerConfig {
     public static final DockerImageName DEFAULT_IMAGE_NAME =
             DockerImageName.parse("crawlercommons/url-frontier");
-    public static final String DEFAULT_TAG = "2.1";
+    public static final String DEFAULT_TAG = "2.2";
```

Comment:
use 'latest' by default?

Reply:
I don't know, I implemented is like a container from the testcontainer repository. It seemed to me like they prefer the usage of a specific version over "latest".

Reply:
> > > Apart from the other comments, could you please break this PR down e.g. separate one for adding the constants, one for reproducing the bug, one for improving the code etc...? thanks
> > 
> > 
> > I could split the PR into three smaller PRs:
> > 
> > 1. Constants + Spout
> > 2. Constants + Status
> > 3. Constants + Tests (This one would fail)
> > 
> > Would that be more convenient for you?
> 
> This would be great. I expect the test to pass now that URLFrontier 2.2 has been released (provided of course that his is the version of the image that gets used). Splitting it would really help as there are quite a few things to explain / discuss. Thanks a lot, it's great to get your contributions on the URLfrontier module.

With version 2.2 everything works as expected.

<img width="395" alt="image" src="https://user-images.githubusercontent.com/15279363/177318998-b2c70cdf-babe-492d-aae4-fa9ec5f6f1e2.png">

Reply:
Thats true, I removed the code specifying the version. This way of declaring testcontainers was deprecated anyway and only kept for backwards compatibility.